specification/netapp/resource-manager/Microsoft.NetApp/preview/2025-07-01-preview/netapp.json

```diff
@@ -0,0 +1,12140 @@
+          "$ref": "#/definitions/EncryptionIdentity",
+          "description": "Identity used to authenticate to KeyVault. Applicable if keySource is 'Microsoft.KeyVault'."
+        }
+      }
+    },
+    "KeyVaultProperties": {
+      "description": "Properties of key vault.",
+      "type": "object",
+      "required": [
+        "keyVaultUri",
+        "keyName"
+      ],
+      "properties": {
+        "keyVaultId": {
+          "type": "string",
+          "description": "UUID v4 used to identify the Azure Key Vault configuration",
+          "readOnly": true,
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+          "maxLength": 36,
+          "minLength": 36,
+          "example": "9760acf5-4638-11e7-9bdb-020073ca7778"
+        },
+        "keyVaultUri": {
+          "type": "string",
+          "format": "uri",
+          "description": "The Uri of KeyVault."
+        },
+        "keyName": {
+          "type": "string",
+          "description": "The name of KeyVault key."
+        },
+        "keyVaultResourceId": {
+          "type": "string",
+          "description": "The resource ID of KeyVault.",
+          "format": "arm-id",
+          "x-ms-arm-id-details": {
+            "allowedResources": [
+              {
+                "type": "Microsoft.KeyVault/vaults"
+              }
+            ]
+          }
+        },
+        "status": {
+          "type": "string",
+          "description": "Status of the KeyVault connection.",
+          "readOnly": true,
+          "enum": [
+            "Created",
+            "Updating",
+            "InUse",
+            "Deleted",
+            "Error"
+          ],
+          "x-ms-enum": {
+            "name": "KeyVaultStatus",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Created",
+                "description": "KeyVault connection created but not in use"
+              },
+              {
+                "value": "InUse",
+                "description": "KeyVault connection in use by SMB Volume"
+              },
+              {
+                "value": "Deleted",
+                "description": "KeyVault connection Deleted"
+              },
+              {
+                "value": "Error",
+                "description": "Error with the KeyVault connection"
+              },
+              {
+                "value": "Updating",
+                "description": "KeyVault connection Updating"
+              }
+            ]
+          }
+        }
+      }
+    },
+    "EncryptionIdentity": {
+      "description": "Identity used to authenticate with key vault.",
+      "type": "object",
+      "properties": {
+        "principalId": {
+          "type": "string",
+          "description": "The principal ID (object ID) of the identity used to authenticate with key vault. Read-only.",
+          "readOnly": true
+        },
+        "userAssi
```

Comment:
Why this get changed?
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/netapp/resource-manager/Microsoft.NetApp/stable/2025-06-01/netapp.json#L9709-L9726